build-scripts/rcm-guest/publish-oc-binary.sh

```diff
@@ -12,13 +12,13 @@ extract() {
     mkdir macosx windows
     for arch in x86_64 ${ARCH}; do
         rpm=$(echo "$(rpm_name "${arch}")"*)
-        [ "${arch}" != x86_64 -a ! -e "${rpm}" ] && continue
+        if [[ "${arch}" != x86_64 ]] && [[ ! -e "${rpm}" ]]; then continue; fi
```

Comment:
`if [[ "${arch}" != x86_64 `**&&**` ! -e "${rpm}" ]]` should work

Reply:
Updated.